src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -40,7 +40,11 @@ import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
 import useGroupInfo from '../GroupInfo/hooks/useGroupInfo';
 
 export const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: dimen.normalizeDimen(40)},
+  container: {
+    flex: 1,
+    backgroundColor: COLORS.almostBlack,
+    paddingBottom: dimen.normalizeDimen(40)
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` property in the `container` style might significantly alter the appearance of the screen. Ensure this color change aligns with the overall design and accessibility guidelines, as darker backgrounds can sometimes reduce readability.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->